### PR TITLE
Fixed state management for Admins

### PR DESCRIPTION
**Pull request by JanviMahajan14**
Initial setup for Signup/Login in hotel-admin-frontend

### CODE REVIEW

**Comment by JanviMahajan14** [comment]
Hi @ayan-biswas0412  I have done the initial set up for the admin website along with state management with Context Api. Do let me know your thoughts on it. we can add components like navbar, loginPage, SignUp page.

File: hotel-admin-frontend/src/App.js [hunk 1]
```diff
@@ -1,14 +1,43 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
-import { HomePage} from './pages';
+import React,{ useEffect, createContext, useReducer, useContext } from 'react';
+import { BrowserRouter, Switch, Route, useHistory } from "react-router-dom";
+import  Home from './pages/Home';
+import  Signup  from './pages/Signup';
```
Comment (ayan-biswas0412) [inline hotel-admin-frontend/src/App.js:4]: rather importing the pages one by one please maintain a index file under /pages and export all them there and import it in a single line , see the directory hotelontouch

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
CI test is failing in one step fix that 
and all other things seems fine

File: hotel-admin-frontend/src/App.js (hunk 1 above)
Comment (JanviMahajan14) [reply hotel-admin-frontend/src/App.js:4]: Sure ! Could you tell me how to fix CI test

Comment (ayan-biswas0412) [reply hotel-admin-frontend/src/App.js:4]: click on the failed test part you will get instructions about what you did wrong

**Comment by aniketsingh98571** [comment]
@JanviMahajan14  `'userState' is assigned a value but never used. lines of code = 1
Found in hotel-admin-frontend\src\App.js:13
13
  const { userState, dispatch } = useContext(userContext);
 React Hook useEffect has missing dependencies: 'dispatch' and 'history'. Either include them or remove the dependency array. lines of code = 1
Found in hotel-admin-frontend\src\App.js:21
21
  }, []);`
Here is the error which you are getting..

**Pull request closed without merging**